### ISSUE TITLE
bootstrap_messages  templatetag does not render safe strings correctly

### ISSUE THREAD

**Issue by mick88** [zostera/django-bootstrap5#140]
Message containing hyperlinks is rendered with html being escaped.

**Comment by dyve** [comment]
Can you add a code example?

**Comment by mick88** [comment]
Add message inside view using django messaging framework
```python
messages.success(request, mark_safe("Click <a href='https://www.github.com/'>here</a>"))
```

Render messages in the template:
```html
{% bootstrap_messages %}
```

**Comment by mick88** [comment]
The MR #142 fixes the issue for me on Django 3.2
